auditlog/models/rule.py

```diff
@@ -507,20 +547,25 @@ def create_logs(
         model_id = self.pool._auditlog_model_cache[res_model]
         auditlog_rule = self.env["auditlog.rule"].search([("model_id", "=", model_id)])
         fields_to_exclude = auditlog_rule.fields_to_exclude_ids.mapped("name")
+
+        vals = {
+            "model_id": model_id,
+            "method": method,
+            "user_id": uid,
+            "http_request_id": http_request_model.current_http_request(),
+            "http_session_id": http_session_model.current_http_session(),
+        }
+        vals.update(additional_log_values or {})
+        if method == "export_data":
+            vals.update({"res_ids": str(res_ids)})
```

Comment:
Conceptual issue: if the ids that were exported are logged, but a record is deleted later on, you don't know what was exported. It would be nice (perhaps later, eg in roadmap) to be able to log also "something about the record" eg the display_name or such.

As a workaround, people could just add an unlink auditlog as well.